administrator/language/en-GB/plg_sampledata_blog.ini

```diff
@@ -24,15 +24,18 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_4_TITLE="Your Modules"
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_6_INTROTEXT="<p>Millions of Websites are built on Joomla!</p>"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_6_FULLTEXT="Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est "
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_6_INTROTEXT="<p><strong>Millions of Websites are built on Joomla!</strong></p><p>Learn more about beautiful blog presentation.</p>"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_6_TITLE="Millions"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_FULLTEXT="<p>Uurnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper artichoke.</p> "
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_INTROTEXT="<p>We love Joomla to the moon and back! Thank you to all volunteers who have contributed!</p> "
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_INTROTEXT="<p><strong>We love Joomla to the moon and back!</strong><p><p> Thank you to all volunteers who have contributed!</p> "
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_7_TITLE="Love"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_FULLTEXT="<p>Cupcake ipsum dolor. Sit amet cotton candy ice cream sesame snaps cake marshmallow powder. Ice cream chocolate cake marshmallow halvah bonbon. Dragée carrot cake danish candy muffin brownie. Candy sugar plum ice cream chupa chups macaroon tiramisu soufflé oat cake. Topping cheesecake lollipop gummi bears icing sweet roll donut liquorice. Pie jelly-o candy donut oat cake cotton candy. </p>"
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_INTROTEXT="<p>We proudly present Joomla Version 4!</p>"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_INTROTEXT="<p><strong>We proudly present Joomla Version 4!</strong></p></p>Learn more about workflows in Joomla.</p>"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_TITLE="Joomla"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_FULLTEXT=""
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Don't panic - your blog entries are not yet published.<br>We have activated the Workflow for the Sample Data Blog.<br>Workflows help you managing the way of your articles have to go until they are published.<br>You first must publish your articles! Go into your Backend and let go your articles through the different stages of your workflow until they are published. </p>"
```

Comment:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Don't panic - your blog entries haven't been published yet.<br>We have activated the Workflow for the Sample Data Blog.<br>Workflows help you managing the way of your articles have to go until they are published.<br>You first must publish your articles! Go into your backend and let go your articles through the different stages of your workflow until they are published. </p>"
```

Reply:
>Workflows help you managing the way of your articles have to go until they are published.

==>

>Workflows manages the stages your articles must go through until they are published

> Go into your backend and let go your articles through the different stages 

In the administrator process your articles through the different stages